scripts/includes/common.sh

```diff
@@ -173,7 +173,12 @@ generate_symmetric_crypto_key_for_datastore() {
   DATASTORE_ENCRYPTION_KEY_PATH="${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}/datastore_key.json"
 
   sudo mkdir -p ${DATASTORE_ENCRYPTION_KEYS_DIRECTORY}
-  sudo st2-generate-symmetric-crypto-key --key-path ${DATASTORE_ENCRYPTION_KEY_PATH}
+
+  # If the file ${DATASTORE_ENCRYPTION_KEY_PATH} exists and is not empty, then do not generate
+  # a new key. st2-generate-symmetric-crypto-key fails if the key file already exists.
+  if [ ! -s ${DATASTORE_ENCRYPTION_KEY_PATH} ]; then
+    sudo st2-generate-symmetric-crypto-key --key-path ${DATASTORE_ENCRYPTION_KEY_PATH}
+  fi
 
   # Make sure only st2 user can read the file
   sudo usermod -a -G st2 st2
```

Comment:
This line could be removed, as we did in `st2docs` just a few days before: https://github.com/StackStorm/st2docs/pull/620

> `st2` user is already part of `st2` group, added by [`deb`](https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/preinst#L36) and [`rpm`](https://github.com/StackStorm/st2-packages/blob/master/packages/st2/rpm/preinst_script.spec#L21) packaging.